scripts/nuget-push.sh

```
@@ -0,0 +1,105 @@
+  exit 1
+fi
+
+# Check if packages directory exists
+if [ ! -d "$OUTPUT_DIR" ]; then
+  echo -e "${RED}Error: Package directory not found: $OUTPUT_DIR${NC}"
+  echo -e "${YELLOW}Run ./nuget-pack.sh first to create packages${NC}"
+  exit 1
+fi
+
+# Find all .nupkg files (excluding symbol packages)
+PACKAGES=($(find "$OUTPUT_DIR" -name "*.nupkg" ! -name "*.snupkg" -type f))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)